modules/distr-tracing-tempo-troubleshoot-collecting-diagnostic-data-from-command-line.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="collecting-diagnostic-data-from-command-line_{context}"]
+= Collecting diagnostic data from the command line
+
+When submitting a support case, it is helpful to include diagnostic information about your cluster to Red Hat Support.
+You can use the `oc adm must-gather` tool to gather diagnostic data for resources of various types, such as `TempoStack` or `TempoMonolithic`, and the created resourced like `Deployment`, `Pod`, or `ConfigMap`.
+The `oc adm must-gather` tool creates a new pod that collects these data.
```

Comment:
"these data" sounds wrong, even though I understand it could be plural. Just checked the ISG and it says to use it as singular

![image](https://github.com/user-attachments/assets/9a9b0693-c48c-4541-a46b-d3e811c5f34d)

```suggestion
The `oc adm must-gather` tool creates a new pod that collects this data.
```

Reply:
OK. Changed.